Improve README.md flow (#532)
semgrep/semgrep: README.md

## Usage

### Example Usage

> Let's start with a simple example. We begin with a Python file, `test.py`. We want to retrieve an object by it's ID, and write:

Reviewer: ```suggestion
Here is a simple Python example, `test.py`. We want to retrieve an object by ID:
```
